README.md

```diff
@@ -36,11 +36,16 @@ integration and I’ll mention it here).
 ``` javascript
 var Edited = require(‘edited’)
 
-var cb = function() {
-    // register undo point, autosave, etc
+var onSensible = function () {
```

Comment:
It's not clear from this documentation what the difference is between `onSensible` and `onAny`. What makes it `sensible`?